src/Microsoft.AspNet.Owin/WebSockets/WebSocketAdapter.cs

```diff
@@ -134,7 +134,7 @@ private Task RedirectSendToCloseAsync(ArraySegment<byte> buffer, CancellationTok
                     _webSocket.Abort();
                     break;
                 default:
-                    throw new ArgumentOutOfRangeException("state", _webSocket.State, string.Empty);
+                    throw new ArgumentOutOfRangeException(nameof(_webSocket.State), _webSocket.State, string.Empty);
```

Comment:
This seems like it's not even the right exception type... there's no "argument" (i.e. a method parameter) here.